deposit/forms.py

```diff
@@ -139,5 +140,22 @@ def __init__(self, *args, **kwargs):
             Submit('submit', _('Save')),
         )
 
-
-
+class RepositoryAdminForm(forms.ModelForm):
+    """
+    Changes widget of the protocol field to a select (dropdown) of AdminView of Repository. This way no free has to be entered for the protocol to choose.
```

Comment:
free "text" ? (I think there is a missing word here)

Reply:
Yep, inserted, thanks!